README.md

```diff
@@ -1,3 +1,103 @@
-# NGS_RNA 
+<h1>NGS_RNA pipeline </h1>
 
-https://github.com/gatk-workflows/gatk4-rnaseq-germline-snps-indels/blob/master/gatk4-rna-best-practices.wdl
+<h2>Description of the different steps used in the RNA analysis pipeline </h2>
+
+<h3>Gene expression quantification </h3>
+The trimmed fastQ files where aligned to  a reference genome using Star [1] with default settings. Before gene quantification 
+SAMtools [2] was used to sort the aligned reads. 
+The gene level quantification was performed by HTSeq-count [3] using --mode=union. 
+The gene annotation database which is included in the results dir.
+in folder expression/. Deseq2 was used for differential expression analysis on STAR bams.
+For experimental group conditions the 'condition' column in the samplesheet was used the 
+distinct groups within the samples.
+
+<h3>Calculate QC metrics on raw and aligned data </h3>
+Quality control (QC) metrics are calculated for the raw sequencing data. This is done using 
+the tool FastQC [4]. QC metrics are calculated for the aligned reads using 
+Picard-tools [5] CollectRnaSeqMetrics, MarkDuplicates, CollectInsertSize-
+Metrics and SAMtools flagstat.
+
+<h3>Splicing event calling using LeafCutter</h3>
+Leafcutter quantifies RNA splicing variation detection.
```

Comment:
On line 20, a capital 'C' is used in LeafCutter. In this line, no capital 'C' is used for Leafcutter.